nautobot_golden_config/migrations/0031_goldenconfigsetting_backup_enabled_and_more.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.17 on 2024-12-19 20:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("nautobot_golden_config", "0030_alter_goldenconfig_device"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="goldenconfigsetting",
+            name="backup_enabled",
+            field=models.BooleanField(default=True),
```

Comment:
is there a way we can pull these from the nautobot__config constants values?  I don't think we want to default these to true